nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
To handle the implicit case, there are new options to set credentials to be used when none are specified in the url.  Would we have to mark?  Could this just be documented, where we say something like _"Local credentials will be included if specified by the application in a connection url, but credentials will never be included in server updated urls."_